src/qibocal/protocols/characterization/randomized_benchmarking_unrolling/std_rb_unrolling.py

```diff
@@ -0,0 +1,218 @@
+    """Raw fitted parameters."""
+
+
+StdRBType = np.dtype(
+    [
+        ("sequence", np.int64),
+        ("length", np.int64),
+        ("probabilities", np.float64),
+    ]
+)
+
+
+@dataclass
+class StdRBData(Data):
+    """StdRB data acquisition."""
+
+    sequences: List[PulseSequence] = None
+
+    data: dict[QubitId, npt.NDArray[StdRBType]] = field(default_factory=dict)
+    """Raw data acquired."""
+
+    def register_qubit(self, qubit, length, prob):
+        """Store output for single qubit."""
+        ar = np.empty((1,), dtype=StdRBType)
+        ar["length"] = length
+        ar["probabilities"] = prob
+        if qubit in self.data:
+            self.data[qubit] = np.rec.array(np.concatenate((self.data[qubit], ar)))
+        else:
+            self.data[qubit] = np.rec.array(ar)
+
+
+def _acquisition(
+    params: StdRBParameters, platform: Platform, qubits: Qubits
+) -> StdRBData:
+    """The data acquisition stage of Standard Randomized Benchmarking.
+
+    1. Set up the scan
+    2. Execute the scan
+    3. Post process the data and initialize a standard rb data object with it.
+
+    Args:
+        params (StandardRBParameters): All parameters in one object.
+        platform (Platform): Platform the experiment is executed on.
+        qubits (dict[QubitId, Union[str, int]] or list[Union[str, int]]): list of qubits the experiment is executed on.
+
+    Returns:
+        RBData: The depths, samples and ground state probability of each experiment in the scan.
+    """
+
+    # Circuit depths
+    depths = list(range(params.min_depth, params.max_depth, params.step_depth))
+
+    # create sequences of pulses for the experiment
+    rb_sequencer = utils.RBSequence(
+        platform,
+        depths,
+        params.runs,
+    )
+
+    # create a Data object to store the results
+    data = StdRBData()
+
+    # Execute the scan
+    for qubit in qubits.values():
+        # create the list of sequences for the experiment
+        sequences, circuits, ro_pulses = rb_sequencer.get_sequences_list(qubit.name)
+        results = platform.execute_pulse_sequences(
+            sequences,
+            ExecutionParameters(
+                nshots=params.nshots,
+                relaxation_time=params.relaxation_time,
+                acquisition_type=AcquisitionType.DISCRIMINATION,
+                averaging_mode=AveragingMode.CYCLIC,
+            ),
+        )
+
+        # count the ro_pulses repeated serials per depth
+        counts_depths = {}
+        for depth in depths:
+            starts = [pulse.start for pulse in ro_pulses[depth]]
+            counts = {}
+            for pulse in ro_pulses[depth]:
+                counts[pulse] = starts.count(pulse.start)
+            counts_depths[depth] = counts
+
+        # average msr, phase, i and q over the number of shots defined in the runcard
+        j = 0
+        for depth in depths:
+            for ro_pulse in counts_depths[depth].keys():
```

Comment:
```suggestion
            for ro_pulse in counts_depths[depth]:
```
It is exactly the same